<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1037</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1037</width>
    <height>540</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>50</green>
        <blue>76</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/mainLogo/resources/5.png</normaloff>:/mainLogo/resources/5.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {background-color: rgb(38, 50, 76);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxArrivalCity">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDepartureCity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditArrivalCity">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 10pt &quot;Britannic Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDepartureCity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
border: rgb(84, 96, 117);
background-color: rgb(67, 81, 108);
font: 10pt &quot;Britannic Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditDeparture">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Britannic Bold&quot;;
background-color: rgb(232, 232, 232);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditBack">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);
font: 10pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTwoWays">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowCalculator">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 13pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frameCalculator">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonMinusKids">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelChildrenCount">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelAdultsCount">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pictures.qrc">:/people/resources/people/2/Adult.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelKidsCount">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pictures.qrc">:/people/resources/people/2/Child.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButtonPlusChildren">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonMinusChildren">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelChildren">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Children</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonMinuseAdults">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButtonPlusKids">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelKids">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Kids</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonPlusAdults">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBoxTicketType">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="title">
         <string>Class</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>311</width>
           <height>27</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonEconomy">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 170, 127);
font: 10pt &quot;Britannic Bold&quot;;</string>
            </property>
            <property name="text">
             <string>Economy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBusiness">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 170, 127);
font: 10pt &quot;Britannic Bold&quot;;</string>
            </property>
            <property name="text">
             <string>Business</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pictures.qrc">:/people/resources/people/2/Kid.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelAdults">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);
font: 8pt &quot;Britannic Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Adults</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonFind">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLogOut">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>500</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 10pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>LogOut</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
font: 12pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>751</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/mainLogo/resources/10.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdminPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(67, 81, 108);
font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>AdminPanel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
